test/e2e/edgedevice_test.go

```diff
@@ -167,8 +170,19 @@ func (e *edgeDeviceDocker) DumpLogs(extraCommands ...string) {
 		if err != nil {
 			fmt.Printf("Error: Failed to retrieve logs for command '%s': %v \n", cmd, err)
 		}
-		ginkgo.GinkgoT().Logf("Command: %s \n Output:\n %s\n", cmd, output)
+		logsMap[cmd] = output
+	}
+
+	return logsMap, err
+}
 
+func (e *edgeDeviceDocker) DumpLogs(extraCommands ...string) {
+	logsMap, err := e.GetLogs(extraCommands...)
+	if err != nil {
+		fmt.Printf("Error: GetLogs failed: %v \n", err)
```

Comment:
Can we replace all `fmt.Printf` with `ginkgo.GinkgoT().Logd` ?

Reply:
Done